sharepa/analysis.py

```diff
@@ -36,7 +37,9 @@ def agg_to_two_dim_dataframe(agg):
         if dict not in [type(item) for item in bucket_as_dict.values()]:
             return bucket_to_dataframe('doc_count', agg.buckets)
         else:
-            name_of_lower_level = bucket_as_dict.keys()[0]
+            lower_level_dict = [item for item in bucket_as_dict.keys() if type(bucket_as_dict[item]) is dict]
+            assert len(lower_level_dict) == 1
```

Comment:
Best to raise rather than assert, asserts can actually be disabled and therefore ignored in production environments.